src/local-notifications-common.ts

```diff
@@ -157,6 +183,16 @@ export interface ScheduleOptions {
    */
   bigTextStyle?: boolean;
 
+  /**
+   * Enable the notification LED light with optional LED light style settings
+   * - empty object if you want to use default settings
+   * - or override the settings specifically
+   *
+   * Android only.
+   * Default not set.
+   */
+  notificationLed?: NotificationLedSettings;
```

Comment:
Not sure if this API is what we want or if we would want to allow the user to just put down a `true` if the user is happy with the default settings (i.e. similar to the `thumbnail` option)
```suggestion
  notificationLed?: boolean | NotificationLedSettings;
```

Of course we would have to update the corresponding usage of this as well :-)